packages/cli/src/create.ts

```diff
@@ -40,6 +40,10 @@ import * as ucc from "./ucc";
 import * as util from "./util";
 import * as schemas from "./schemas";
 import { BUILD_INFO } from "@opstrace/buildinfo";
+import {
+  OPSTRACE_DOCKERHUB_TOKEN,
+  OPSTRACE_DOCKERHUB_USERNAME
```

Comment:
confused that this is imported here but seemingly not used in this module. why does that compile?

Reply:
oh wow, `OPSTRACE_DOCKERHUB_USERNAME` is the name of the environment variable holding the `OPSTRACE_DOCKERHUB_USERNAME`. Can we maybe work on naming here? :D

Reply:
Retyping the string literal is more prone to error though. I much prefer a `const` that can be reused!

Reply:
Can rename it if it helps

Reply:
actually, second thoughts, I think renaming it causes more confusion because then it's easy to mistake the name of the const for the name of the env var.

Reply:
>  I think renaming it causes more confusion because then it's easy to mistake the name of the const for the name of the env var.

Not when using something expressive like `const DOCKERHUB_TOKEN_ENVVAR_NAME` :D (this is really what I was aiming for, should have said that).

well. We will see. :D It was certainly fine to not act on this.